WalletConnect/Models/Trust/WCTrustAccount.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  WCTrustAccount.swift
+//  WalletConnect
```

Comment:
let's use Trust license header

Reply:
We should add this configuration to be used by everyone: https://oleb.net/blog/2017/07/xcode-9-text-macros/

Reply:
sounds good